app/modals/ViewEvent/index.js

```diff
@@ -15,6 +15,8 @@ const ViewEvent = ({ id, slug, onClose, event, startPolling, stopPolling }) => {
     return () => stopPolling();
   }, [id]);
 
+  console.log(event);
```

Comment:
remove

Reply:
yeah i was just testing , forgot to remove.